lib/auth/aad.ts

```diff
@@ -3,7 +3,7 @@
 
 import {
   ApplicationTokenCredentials, DeviceTokenCredentials, UserTokenCredentials, MSITokenCredentials
-} from "ms-rest-azure";
+} from "@azure/ms-rest-nodeauth";
```

Comment:
@bterlson When you tested Service Bus by pointing the amqp-common dependency to your git branch, were you able to run any of the examples like [interactiveLogin](https://github.com/Azure/azure-sdk-for-js/blob/master/packages/%40azure/servicebus/data-plane/examples/javascript/gettingStarted/interactiveLogin.js), [servicePrincipalLogin](https://github.com/Azure/azure-sdk-for-js/blob/master/packages/%40azure/servicebus/data-plane/examples/javascript/gettingStarted/servicePrincipalLogin.js) or the [test on auth](https://github.com/Azure/azure-sdk-for-js/blob/master/packages/%40azure/servicebus/data-plane/test/namespace.spec.ts#L340) without making any changes?

Reply:
@ramya-rao-a I didn't try the samples. I'll report here with my findings, though I may need your help getting the AD side set up properly.